app/src/main/java/com/github/se/orator/model/symblAi/SpeakingRepositoryRecord.kt

```diff
@@ -6,9 +6,10 @@ import java.io.File
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 
-class SpeakingRepositoryRecord(private val context: Context) : SpeakingRepository {
+class SpeakingRepositoryRecord(private val context: Context, private val isOffline: Boolean) :
```

Comment:
I can see how this change might be useful, but maybe add a default value = false in case someone already used this repository but hasn't pushed their changes. We will have to see after I merge offline mode since I also use the repository.